src/app/server/Server.h

```diff
@@ -134,15 +130,24 @@ class Server
         CHIP_ERROR SyncDelete(FabricIndex fabricIndex, const char * key) override { return SyncDeleteKeyValue(key); };
     };
 
+    CHIP_ERROR InitCASESessionManager();
+
 #if CONFIG_NETWORK_LAYER_BLE
     Ble::BleLayer * mBleLayer = nullptr;
 #endif
 
     ServerTransportMgr mTransports;
     SessionManager mSessions;
     CASEServer mCASEServer;
-    CASEClientPoolDelegate * mCASEClientPool         = nullptr;
-    OperationalDeviceProxyPoolDelegate * mDevicePool = nullptr;
+
+    static constexpr size_t kCASEClientPoolSize        = CHIP_CONFIG_DEVICE_MAX_ACTIVE_CASE_CLIENTS;
```

Comment:
nit: We can probably just use `CHIP_CONFIG_DEVICE_MAX_ACTIVE_CASE_CLIENTS` and `CHIP_CONFIG_DEVICE_MAX_ACTIVE_DEVICES` directly on line 148-149